package.json

```diff
@@ -1,18 +1,19 @@
 {
-    "name": "strava-pg",
-    "module": "script.ts",
-    "type": "module",
-    "devDependencies": {
-        "@types/bun": "latest"
-    },
-    "peerDependencies": {
-        "typescript": "5.0.0"
-    },
-    "dependencies": {
-        "@types/prompts": "2.4.9",
-        "pg": "8.11.3",
-        "prettier": "3.2.4",
-        "prompts": "2.4.2",
-        "strava-v3": "2.2.0"
-    }
+  "name": "strava-pg",
```

Comment:
Could you please run prettier on all your files? There's a `.prettierrc` file at the root of the repository to make sure that code styling is consistent.

Reply:
Done!